apis/projectcontour/v1/httpproxy.go

```diff
@@ -294,6 +296,25 @@ type LoadBalancerPolicy struct {
 	Strategy string `json:"strategy,omitempty"`
 }
 
+// RequestHeaderPolicy defines how headers are managed during forwarding
+type RequestHeaderPolicy struct {
+	// ReplaceHeaders defines how headers will be replaced during forwarding
+	// +kubebuilder:validation:Optional
+	ReplaceHeaders []ReplaceHeader `json:"replaceHeaders,omitempty"`
+}
+
+// ReplaceHeaders defines how a header can be replaced during forwarding
+type ReplaceHeader struct {
+	// Key represents a key of a header
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	Key string `json:"key"`
```

Comment:
Suggest that we call this "Name".

Reply:
I don't feel super strong either way, was just going off of @davecheney's suggestion.